app/controllers/placements/providers/placements_controller.rb

```diff
@@ -15,6 +15,11 @@ def index
     )
   end
 
+  def show
+    @placement = Placement.find(params[:id]).decorate
+    @school = @placement.school.decorate
```

Comment:
I wonder if it'd be handy to use Draper's [decorates_association](https://github.com/drapergem/draper?tab=readme-ov-file#decorating-associated-objects) for this?

```ruby
class SchoolDecorator < OrganisationDecorator
  decorates_association :school
end
```

I think this would mean `@placement.school` is automatically decorated.